### PR TITLE
SamplesAndMT2

### PR DESCRIPTION
**Pull request by mariadalfonso**

### CODE REVIEW

**Comment by mariadalfonso** [comment]
@gpetruc added cross section to the tree for the know samples

File: CMGTools/TTHAnalysis/python/samples/ComponentCreator.py [hunk 1]
```diff
@@ -67,6 +65,7 @@ def getFilesFromEOS(self,name,dataset,path,pattern=".*root"):
                 raise RuntimeError, "ERROR making component %s: no files found under %s matching '%s'" % (name,path,pattern)
             writeDatasetToCache('EOS%{path}%{pattern}.pck'.format(path = path.replace('/','_'), pattern = pattern), files)
         return files
+
     def makeMCComponentFromEOS(self,name,dataset,path,pattern=".*root"):
```
Comment (gpetruc) [inline CMGTools/TTHAnalysis/python/samples/ComponentCreator.py]: Since you are at it, you could add the xSec argument also for this one?

**Comment by gpetruc** [comment]
Also, am I missing something or you're not propagating the cross section to the output tree yet?

**Comment by mariadalfonso** [comment]
@gpetruc added the added the xSec also for makeMCComponentFromEOS, added to the xSec into the tree

**Pull request merged**